app/src/main/java/com/amaze/filemanager/utils/ServiceWatcherUtil.java

```diff
@@ -25,26 +30,27 @@
     private Handler handler;
     private static HandlerThread handlerThread;
     private ProgressHandler progressHandler;
-    long totalSize;
     private Runnable runnable;
 
+    public static int STATE = -1;
+
     private static ArrayList<Intent> pendingIntents = new ArrayList<>();
 
     // position of byte in total byte size to be copied
-    public static long POSITION = 0L;
+    public static volatile long POSITION = 0L;
+
+    private static int HALT_COUNTER = -1;
 
-    private static int HAULT_COUNTER = -1;
+    public static final int ID_NOTIFICATION_WAIT =  9248;
```

Comment:
Shouldn't this go in `NotificationConstants`?

Reply:
Yes, will do the needful.